compat/beaver-builder/styles.less

```diff
@@ -193,3 +193,22 @@
 .so-widgets-dialog .so-widgets-search-input {
 	width: 100%;
 }
+
+.fl-builder-ui-skin--dark {
+	.siteorigin-widget-form {
+		 .siteorigin-widget-field-repeater .siteorigin-widget-field-repeater-top, .siteorigin-widget-field-repeater .siteorigin-widget-field-repeater-items .siteorigin-widget-field-repeater-item, .siteorigin-widget-field-repeater .siteorigin-widget-field-repeater-items .siteorigin-widget-field-repeater-item .siteorigin-widget-field-repeater-item-top, .siteorigin-widget-field-repeater .siteorigin-widget-field-repeater-items .siteorigin-widget-field-repeater-item .siteorigin-widget-field-repeater-item-form, .siteorigin-widget-field-repeater .siteorigin-widget-field-repeater-items, .siteorigin-widget-field-type-widget > label, .siteorigin-widget-field-type-section > label, .siteorigin-widget-field-type-icon .siteorigin-widget-icon-selector-current, .siteorigin-widget-field-type-icon .siteorigin-widget-icon-selector {
```

Comment:
Could you add line breaks between selectors after the `,` to avoid having such a long line? And the same for the long lines below.